plutus-core/plutus-ir/test/transform/inline/letTypeApp2

```diff
@@ -1,21 +1,16 @@
 {- 
 funApp gets inlined because it only occurs once.
-k should not get inlined by the unconditional inline optimization because it occurs more
-than once.
-Both instances of k should get inlined by the optimization that inlines fully applied functions 
-because it is fully applied.
-The first instance of k being inlined verifies that an over-applied function still gets inline with
-type applications.
+Both instances of k get inlined by unconditional inliner.
+The first instance of k being inlined verifies that the inliner works with type applications.
 
-
-let
+let 
     idFun :: forall a . a -> a
     idFun = \x -> x
     funApp :: Integer -> Integer
     funApp = idFun @Integer
     k :: forall a . a -> a
     k = idFun
-in k @Integer (k @(Integer -> Integer) funApp 3)
+in k @Integer (k @Integer (funApp 3))
```

Comment:
What's the reason for this change?

Reply:
Frankly I can't remember, I think I wanted to make it type check, since `funApp 3` is an `Integer`.